Classes/Utility/Page.php

```diff
@@ -36,16 +36,19 @@ public static function extendPidListByChildren($pidList = '', $recursive = 0)
             return $pidList;
         }
 
+        /** @var \TYPO3\CMS\Core\Database\QueryGenerator $queryGenerator */
```

Comment:
please remove this line, those are not needed - use the phpstorm plugin `DynamicReturnTypePlugin`